_posts/2017-08-18-forbes.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Forbes
+sourceURL: https://digiday.com/media/new-mobile-site-forbes-boosted-impressions-per-session-10-percent/
+image: /images/forbes.png
+tags:
+- News
+- Notifications
+- Offline
+- Service Worker
+---
+
+**Forbes’** PWA loads in **2.5** seconds on mobile compared to 6.5 seconds for its previous site. Impressions per visit are up **10%**.
```

Comment:
That's a question I probably can't answer directly. Maybe I can try to explain my git experience. 

1. For some reason, I didn't see this branch locally anymore when typing `git branch`
2. I did a git fetch origin - still didn't see it.
3. I executed git checkout `this branch` and checked out the branch
4. I ran `git pull origin master` got a warning about non fast forward merging (I think). 
5. I made changes, commited them and tried to push. 
6. Git said I could not push my changes because "Updates were rejected because the tip of your current branch is behind"
7. I did a git pull.
8. I got conflicts in `head.html` - fixed conflicts. Commited changes
9. Pushed branch back up to origin.